packages/cloud_firestore_odm/cloud_firestore_odm/example/lib/integration.dart

```diff
@@ -22,6 +22,7 @@ class EmptyModel {
 final emptyModelRef = EmptyModelCollectionReference();
 
 @Collection<ManualJson>('root')
+@JsonSerializable()
```

Comment:
That defeats the point of this test. This test is about _not_ using json_serializable

Reply:
Yes